include/albatross/NullModel

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2019 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef ALBATROSS_NULL_MODEL_H
+#define ALBATROSS_NULL_MODEL_H
```

Comment:
This doesn't have mark up, probably because the file doesn't have an extension - it should be a .h, right?

Reply:
Actually, `albatross` has been using the `Eigen` style of includes, ie individual header files don't include anything and then a centralized include file (`#include <albatross/NullModel>`) deals with knowing what needs to be included and in what order.

Reply:
The rest of the centralized includes are actually extension free as well.  This has confused other people too, so could be worth switching, but does match `Eigen`, (`#include <Eigen/Core>` and `#include <albatross/Core>`).